client/go/cmd/clone.go

```diff
@@ -148,17 +135,50 @@ func getSampleAppsZip() (*os.File, error) {
 			return fmt.Errorf("could not download sample apps: %w", err)
 		}
 		defer response.Body.Close()
-		if response.StatusCode != 200 {
+		if response.StatusCode != http.StatusOK {
 			return fmt.Errorf("could not download sample apps: github returned status %d", response.StatusCode)
 		}
-		if err := f.Truncate(0); err != nil {
-			return fmt.Errorf("could not truncate sample apps file: %s: %w", f.Name(), err)
-		}
 		if _, err := io.Copy(f, response.Body); err != nil {
 			return fmt.Errorf("could not write sample apps to file: %s: %w", f.Name(), err)
 		}
+		if err := os.Rename(f.Name(), destination); err != nil {
```

Comment:
`f` needs to be closed before moving (Linux/macOS is fine with this, but Windows isn't).